src/components/MemberArea/tags.js

```diff
@@ -28,13 +28,17 @@ export default [
   'build',
   'c',
   'c++',
+  'c++/cli',
+  'c++/winrt',
   'c-sharp',
   'career',
+  'ci',
   'cloud',
   'codeigniter',
   'command-line',
   'compassion',
   'computability',
+  'continuous integration',
```

Comment:
Isn't synonyms of `ci`? If so, we prefer to hold only unique tags so people will not have to guess about the meaning and the difference between the tags.

Reply:
it is, ci should probably be removed indeed

Reply:
```suggestion
```